.tours/codeql-tutorial.tour

```diff
@@ -4,20 +4,29 @@
   "isPrimary": true,
   "steps": [
     {
-      "description": "First let's talk about the CodeQL CLI.\n\nThe CodeQL extension uses the CodeQL CLI to compile and run queries. It automatically manages access to the executable of the CLI for you.\n\nWe've already set it up to use in this tutorial. To learn more about the CodeQL CLI, see [the docs](https://codeql.github.com/).",
+      "description": "First thing's first, let's get some setup out of the way.\n\nIn the next step you will be prompted to open the tutorial workspace and allow telemetry.\n\nOnce you've done this, please return to this step in the Code Tour so we can explore the CodeQL extension.\n\n Now please click 'Next (The CodeQL Extension)'.",
```

Comment:
I don't understand this new text. I ran it in the code tour directly to see if I was missing something but it is still confusing me.
It sounds like the user should take some action here? Is there something the user needs to do before `Now please click ..` ?

Reply:
Ah, good point ❤️ Instead of the last two lines (`Once you've done this, return to this step in the code tour so we can explore the CodeQL extension.\n\n Now please click 'Next (The CodeQL extension)'.`), maybe we can just say something like `To get started, click 'Next (The CodeQL extension)'.`

Would that be clearer?